site/src/app/InstructionsDialog.tsx

```diff
@@ -44,7 +44,7 @@ function InstructionsDialog({ app, close }: Props): JSX.Element {
 
                     <CodeBlock
                         text={`west init -m "${app.repo}" --mr ${
-                            latestRelease?.tag ?? '<latest tag>'
+                            latestRelease ?? '<latest tag>'
```

Comment:
With the changed assignment above `latestRelease` will always be defined, so the `?? '<latest tag>'` could be removed here. But it is also ok if you want to keep it as an additional safety measure.